shared/charts/couchdb-prometheus-exporter/templates/deployment.yaml

```diff
@@ -10,7 +10,7 @@ spec:
       labels:
         app: couchdb-prometheus-exporter
       annotations:
-        sidecar.istio.io/inject: "false"
+        traffic.sidecar.istio.io/excludeOutboundIPRanges: "169.254.169.254/32"
```

Comment:
Might be worth mentioning what this is for?

If so, please update the same line in the flowmanager and preferences charts.

Reply:
This is to allow metadata traffic. Once we enable sidecar auto-injection for couchdb-prometheus-exporter, all destinations that are not explicitly allowed with ServiceEntries, are forbidden. And Istio 1.0.6 is still affected by FQDN issue (https://github.com/istio/istio/issues/14404), so allowing metadata traffic is impossible via ServiceEntry, hence this hack.

Reply:
Great, thanks. Please add that explanation as a comment to all three charts.